build.gradle

```diff
@@ -0,0 +1,107 @@
+buildscript {
+  ext.kotlin_version = "[1.2,)"
+
+  dependencies {
+    repositories {
+      mavenCentral()
+      jcenter()
```

Comment:
This might be obvious but I'm not certain why you need both `mavenCentral` and `jcenter` here?

Reply:
`jcenter` is required to resolve the `dokka` plugin